### ISSUE TITLE
Interop tests are failing

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#4891]
Many cloud to cloud and cloud to prod tests are failing. Interop test runs take >1h.

https://grpc-testing.appspot.com/job/gRPC_interop_master/2594/

**Comment by jtattermusch** [comment]
@yang-g CC

**Comment by murgatroid99** [comment]
Most of those tests have output indicating success, but they also timed out. Maybe the worker got starved out.

**Comment by adelez** [comment]
This is because ESF has a health checking bug when we enable port sharing for serving both Autobahn and gRPC over LOAS2. A fix will come in soon.

**Comment by adelez** [comment]
Restarted ESF with the fix. Should work now.

**Comment by adelez** [comment]
Leaving this config for some more time for @yang-g to debug. If we don't find a solution today, we'll modify the config. Currently all interop tests time out.

**Comment by jtattermusch** [comment]
Building all interop docker images is failing with:
`Error response from daemon: Error running deviceCreate (createSnapDevice) dm_task_run failed`
https://grpc-testing.appspot.com/job/gRPC_interop_master/2607/console

**Comment by jtattermusch** [comment]
something badly wrong with the grpc-canary-interop1

Even after restart, I see
`jenkins@grpc-canary-interop1:/home/jtattermusch$ docker run -it debian:jessie /bin/bash
Error response from daemon: Error running deviceCreate (createSnapDevice) dm_task_run failed
`

I'll probably nuke the VM and recreate it from scratch.

**Comment by jtattermusch** [comment]
I recreated grpc-canary-interop1 from scratch.

**Comment by adelez** [comment]
Thank you, Jan!

**Comment by adelez** [comment]
I rebuilt the backend with some fixes. Local tests show success. We should see success on Jenkins starting from now.

**Comment by jtattermusch** [comment]
Good work, let's see if the tests start to succeed.

**Comment by jtattermusch** [comment]
Some problems building the java image: https://github.com/grpc/grpc-java/issues/1370

**Comment by jtattermusch** [comment]
On friday, the last update was that most of the tests were actually succeeding, but after they exited, the docker container continued running, which eventually resulted in a timeout.

Today morning, I switched the storage option for all jenkins workers to overlayFS (it was devicemapper before, see #4988). Finally, the interop tests seem to be passing again.

https://grpc-testing.appspot.com/job/gRPC_interop_master/2706/

Closing this for now, let's reopen if we see major interop tests problems during the day.
